src/Autoloader.php

```diff
@@ -94,7 +94,12 @@ private static function createPrependAutoloader(array $namespaces, ClassLoader $
             }
 
             if ($classLoader->loadClass($class)) {
-                $legacy = $namespaces[$check] . str_replace('Laminas', 'Zend', substr($class, strlen($check)));
+                $legacy = $namespaces[$check]
+                    . strtr(substr($class, strlen($check)), [
+                        'ApiTools' => 'Apigility',
+                        'Mezzio' => 'Expressive',
+                        'Laminas' => 'Zend',
```

Comment:
we need unit tests for all these, as I've tried to comment it out each of above line any all tests still were fine